frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -22,6 +22,7 @@ import {
 import { RegistryType } from '../../utils/imagestream-utils';
 import { AppResources } from '../edit-application/edit-application-types';
 import { DeployImageFormData, Resources } from './import-types';
+import { getRandomChars } from '../pipelines/pipeline-resource/pipelineResource-utils';
```

Comment:
We need to be putting this function in a nicer location... import for knative pulling from pipelines is odd.

We can probably leave this for the sake for 4.4 getting done but we should log a refactor task to clean this up.

We could probably keep this to `dev-console` though...

Reply:
can we have it in under `devcnsole/utils` in `shared-submit-utils.ts` ?

Reply:
Yeah, that's works